openghg/util/_domain.py

```diff
@@ -150,12 +150,30 @@ def convert_longitude(longitude: ArrayLikeMatch) -> ArrayLikeMatch:
     Returns:
         ndarray / DataArray : Updated longitude values in the same order.
     """
-    # Check range of longitude values and convert to -180 - +180
+
+    # Check range of longitude values and convert to -180 to 180.
     longitude = ((longitude - 180) % 360) - 180
 
     return longitude
 
 
+def convert_lon_to_360(longitude: ArrayLikeMatch) -> ArrayLikeMatch:
+    """
+    Convert longitude extent to (0 to 360).
+    This does *not* reorder the values.
+
+    Args:
+        longitude: Valid longitude values in degrees.
+    Returns:
+        ndarray / DataArray : Updated longitude values in the same order.
+    """
+
+    # Check range of longitude values and convert to 0 to 360.
+    longitude = (longitude + 360) % 360
```

Comment:
```suggestion
    longitude = longitude % 360
```
Since 360 % 360 is 0, we can leave that part out.